CHANGELOG.md

```diff
@@ -13,6 +13,9 @@
 ## New features
 - New feature to omit the `source_column_name` column on the `union_relations` macro ([#331](https://github.com/dbt-labs/dbt-utils/issues/331), [#624](https://github.com/dbt-labs/dbt-utils/pull/624))
 
+## Under the hood
```

Comment:
I think you'll need to merge the head of `utils-v1` branch in to avoid merge conflicts with other changelog items that have already been made